sktime/forecasting/base/adapters/_pmdarima.py

```diff
@@ -66,9 +66,18 @@ def _predict(self, fh, X=None, return_pred_int=False, alpha=DEFAULT_ALPHA):
 
         # both in-sample and out-of-sample values
         else:
-            y_ins = self._predict_in_sample(fh_ins, **kwargs)
-            y_oos = self._predict_fixed_cutoff(fh_oos, **kwargs)
-            return y_ins.append(y_oos)
+            if return_pred_int:
```

Comment:
what are you doing here? This does not look correct.
Sorry for not noticing earlier.

The return type of `self._predict_in_sample(fh_ins, **kwargs)` does not depend on the value of `return_pred_it` - this will break if it's `True`!

It's also unrelated to the change you wanted to make - I'd recommend to simply revert this.

Reply:
I tried solving this issue #1441  at the same time since I can't do tests properly with it raising an error in every attempt that tuple append.

Reply:
ahhhhhh - that's why!

I guess it's fine as a temporary, hacky fix then, but it should not stay this way for too long.

Would you be sticking around to fix it completely, after #1421 is merged?
Might be good to have your eyes on this since you've seen the internals.